### DISCUSSION TITLE
每个人都应该拥有一个属于自己的网站

### DISCUSSION THREAD

**Discussion by giscus** [ceynri/ceynri.cn#38] [Comments]
# 每个人都应该拥有一个属于自己的网站

山风的小角落 / Ceynri's personal website

https://ceynri.cn/blog/v2-ceynri-cn-build-summary/

**Comment by xiamuguizhi** [comment]
很酷！

**Comment by miaopasixx** [comment]
界面很美，但是我进入每篇文章不知道怎么回主页。

**Reply by ceynri** [reply]
当时一直在纠结把返回键放在哪里合适，结果就到现在都没有把返回方式给加上😂
目前想要返回的话可以通过浏览器的返回按钮（但是确实不是很友好）
过两周有空了我就把这个功能补上！
